### PR TITLE
Plugins SPA - enabled by default.

### PR DESCRIPTION
**Pull request by varshavaradarajan**
* The old plugins page is available at go/admin/old_plugins

### CODE REVIEW

**Comment by ketan** [comment]
I believe you'd raised some concern about icons for bad plugins, is that logged somewhere?

**Comment by varshavaradarajan** [comment]
https://github.com/gocd/gocd/pull/3932#issuecomment-336365888

File: server/webapp/WEB-INF/rails.new/webpack/views/plugins/plugins_widget.js.msx [hunk 1]
```diff
@@ -87,18 +87,7 @@ const PluginsWidget = {
     };
 
     this.edit = function (pluginInfo) {
-      const newPluginSetting   = Stream(new PluginSetting({pluginId: pluginInfo.id()}));
-      const setPluginSetting = function () {
```
Comment (maheshp) [inline server/webapp/WEB-INF/rails.new/webpack/views/plugins/plugins_widget.js.msx]: Is this behaviour tested, should we change the tests for this?

File: server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/single_page_apps/plugins.scss [hunk 2]
```diff
@@ -106,6 +106,8 @@ $elastic-profile-icon-hover-color: #000;
     text-overflow: ellipsis;
     white-space:   nowrap;
     padding-right: 30px;
+    font-weight: 400;
```
Comment (maheshp) [inline server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/single_page_apps/plugins.scss]: format this

**Pull request merged**